src/BlockchainReader/Bitcoin.ts

```diff
@@ -8,6 +8,18 @@ interface VOutWithTxId extends VOut {
   readonly transactionId: string
 }
 
+enum AnchorSectionsLengths {
+  Prefix = 4,
+  StorageProtocol = 1,
+  Version = 2,
+  IPFSHash = 46,
+}
+const anchorByteLength =
```

Comment:
instead of adding these together manually can we sum the values of the enum?

Reply:
maybe Object.values

Reply:
We could do either(or something like it):
```ts
const AnchorSectionsLengths = {
  Prefix: 4,
  StorageProtocol:  1,
  Version: 2,
  IPFSHash: 46,
}

const add = (a: number, b: number) => a + b
const anchorBytesLength = Object.values(AnchorSectionsLengths).reduce(add)
```
or:
```ts
enum AnchorSectionsLengths {
  Prefix = 4,
  StorageProtocol =  1,
  Version = 2,
  IPFSHash = 46,
}

const addNums = (a: number, b: number) => isNaN(a) || isNaN(b) ? a + 0 : a + b
const anchorByteLength = Object.values(AnchorSectionsLengthss).reduce(addNums, 0)
```

Reply:
Better, it might be a bit nicer using ramda though

https://ramdajs.com/docs/#values
https://ramdajs.com/docs/#sum

```ts
const sumObjectValues = pipe(values, sum)
const anchorByteLength = sumObjectValues(AnchorSectionsLengths)
```